src/components/SingleName/ResolverAndRecords/RecordsItem.js

```diff
@@ -133,22 +167,35 @@ const RecordItemEditable = ({
 }) => {
   const { state, actions } = useEditable()
 
-  const { editing, newValue, txHash, pending, confirmed } = state
+  const {
+    editing,
+    authorized,
+    uploading,
+    newValue,
+    txHash,
+    pending,
+    confirmed
+  } = state
 
   const {
     startEditing,
     stopEditing,
     updateValue,
     startPending,
-    setConfirmed
+    setConfirmed,
+    startUploading,
+    startAuthorizing,
```

Comment:
Don't think it's a good idea to add uploading and authorising to this reducer inside RecordItem as it only applies to Temporal/IPFS. Seems to be getting a bit messy in RecordsItem, so might be worth splitting it into it's own component called ContenthashRecord and have a special useEditable reducer for contenthash with the additional fields. This will stop `loggedIn()` being called for other records like address as well when it initialises another `useEditable`